matminer/featurizers/composition.py

```diff
@@ -1320,3 +1319,123 @@ def citations(self):
 
     def implementors(self):
         return ['Qi Wang', 'Alireza Faghaninia']
+
+
+class YangSolidSolution(BaseFeaturizer):
+    """
+    Mixing thermochemistry and size mismatch terms of Yang and Zhang (2012)
+
+    This featurizer returns two different features developed by
+    .. Yang and Zhang `https://linkinghub.elsevier.com/retrieve/pii/S0254058411009357`.
+    The first, Omega, is related to the balance between the mixing entropy and
+    mixing enthalpy of the liquid phase. The second, delta, is related to the
+    atomic size mismatch betweens the
+
+    Features
+        yang Omega - Mixing thermochemistry feature, Omega
+        yang delta - Atomic size mismatch term
+
+    References:
+        .. Yang and Zhang (2012) `https://linkinghub.elsevier.com/retrieve/pii/S0254058411009357`.
+    """
+
+    def __init__(self):
+        # Load in the mixing enthalpy data
+        #  Creates a lookup table of the liquid mixing enthalpies
+        mixing_dataset = pd.read_csv(os.path.join(data_dir,
```

Comment:
Any reason not to have mixing_dataset be part of "data.py"? 

e.g. you can do:
self.mixing_data = MixingData()

like you did for loading elem_data and Magpie a few lines below